src/print.ts

```diff
@@ -43,7 +43,7 @@ async function print(type: string, uri?: Uri, outFolder?: string) {
         doc.save();
     }
 
-    window.setStatusBarMessage(localize("printing") + ` '${path.basename(doc.fileName)}' ` + localize("to") + ` '${type.toUpperCase()}' ...`, 1000);
+    const statusBarMessage = window.setStatusBarMessage("$(sync~spin)" + localize("ui.exporting.messageExportingInProgress", path.basename(doc.fileName), type.toUpperCase()));
```

Comment:
Usually we put a space after an icon.

Reply:
I can give it a try, but guess that makes no difference.